test/functional/testlib.bash

```diff
@@ -1132,6 +1136,96 @@ assert_not_in_output() {
 
 }
 
+assert_is_output() {
+
+	local expected_output=$1
+	local sep="------------------------------------------------------------------"
+	validate_param expected_output
+
+	if [[ ! "$output" == "$expected_output" ]]; then
```

Comment:
in swupdlib.bash we're filtering some lines from the output. I think this is a good feature to add since there're some lines that aren't very useful to check if the test is successful, like empty lines. Or lines with dots.

Reply:
Yeah, I noticed that from swupdlib.bash, but I'm not sure if I agree with that approach. For example, let's say you are ignoring the dots in your output, that means your output could get full of dots and your tests would not detect that, and it would be very annoying for the end user. I think tests should be as explicit as possible.

Reply:
@castulo while I agree with the statement that `tests should be as explicit as possible` these tests are for the most part not designed to verify prints to stdout. The goal of the output checks is to ensure the correct steps are happening in the correct order. It is useful to ignore blank and progress lines so we don't have to change every test when we change that non-functional output.

Obviously this feature request is not a blocker for this PR.

Reply:
@matthewrsj @otaviobp I added what you guys requested in the latest commit in this PR, whenever you have a chance please take a look at it.